src/osd/OSD.cc

```diff
@@ -10201,7 +10201,7 @@ void OSD::mon_cmd_set_config(const std::string &key, const std::string &val)
 
 bool OSD::unsupported_objstore_for_qos()
 {
-  static const std::vector<std::string> unsupported_objstores = { "filestore" };
+  static const std::vector<std::string> unsupported_objstores = { "filestore", "memstore" };
```

Comment:
```suggestion
  static const std::vector<std::string> unsupported_objstores =
    { "filestore", "memstore" };
```
Nit: The above suggestion is to preserve 80 chars limit per line.

Reply:
Got it, thanks!